### PR TITLE
Use 4 core external build queues

### PR DESCRIPTION
**Pull request by jashook**
This uses the bring your own cloud machine pools for windows and linux external builds.

### CODE REVIEW

File: eng/xplat-job.yml [hunk 1]
```diff
@@ -35,21 +35,36 @@ jobs:
...
+      # Official Build Linux Pool
       ${{ if and(eq(parameters.osGroup, 'Linux'), ne(variables['System.TeamProject'], 'public')) }}:
         name: dnceng-linux-internal-temp
+
       # FreeBSD builds only in the internal project
       ${{ if and(eq(parameters.osGroup, 'FreeBSD'), ne(variables['System.TeamProject'], 'public')) }}:
         name: dnceng-freebsd-internal
+
+      # Public OSX Build Pool
       ${{ if and(eq(parameters.osGroup, 'OSX'), ne(variables['System.TeamProject'], 'public')) }}:
```
Comment (sbomer) [inline eng/xplat-job.yml]: The OSX and Windows comments need to be swapped across public/internal. Could you re-order these to make the public/internal order consistent while you're at it? Thanks!

Comment (jashook) [reply eng/xplat-job.yml]: Will create another pr with these changes.

**Pull request merged**